<launch>
    <env name="ROSCONSOLE_FORMAT" value="[${severity}][${thread}][${node}/${function}:${line}]: ${message}"/>

    <!-- Joystick Device Argument -->
    <arg name="joy_dev" default="/dev/input/js2"/>

    <!-- Joy Node -->
    <node pkg="joy" type="joy_node" name="joy_drive" clear_params="true" output="screen" respawn="true" >
      <param name="dev" value="$(arg joy_dev)" />
      <param name="deadzone" value="0.12" />
      <param name="autorepeat_rate" value="10" />
      <!-- OUTPUT TOPICS -->
      <remap from="joy" to="joy" />
    </node>

    <!-- Teleoperation Node -->
    <node pkg="simple_arm" type="arm_teleop" name="arm_teleop" clear_params="true" output="screen" respawn="true">
      <param name="gripper_speed" type="int" value="5"/>
      <param name="gripper_speed_max" type="int" value="100"/>
      <param name="gripper_speed_min" type="int" value="25"/>
      <!-- INPUT TOPICS -->
      <remap from="joy" to="joy" />
      <!-- OUTPUT TOPICS -->
      <remap from="gripper_pos" to="gripper_pos" />
    </node>
</launch>
